docs/blog/posts/pydantic.md

```diff
@@ -0,0 +1,235 @@
+---
+draft: False
+date: 2023-12-11
+slug: pydantic
+tags:
+  - pydantic
+authors:
+  - jxnl
+---
+
+# Steering Large Language Models with Pydantic
+
+In the past year, significant progress has been made in utilizing large language models. Prompt engineering, in particular, has gained attention, and new prompting techniques are being developed to guide language models toward specific tasks. While many are building chat bots, an even more exciting application is the generation of structured outputs, whether its extracting structured data, augmenting your RAG application, or even generating
```

Comment:
I'm guessing you meant to finish the "or even generating..." at the end here, but otherwise I like this as an intro paragraph. As someone who hasn't worked with LLMs a bunch, it could be helpful to add an additional sentence going into more detail about what "prompt engineering" is.

Reply:
added!